packages/twenty-front/src/modules/object-metadata/utils/formatFieldMetadataItemInput.ts

```diff
@@ -32,14 +32,14 @@ export const formatFieldMetadataItemInput = (
     const defaultOptions = options?.filter((option) => option.isDefault);
     if (isDefined(defaultOptions)) {
       defaultValue = defaultOptions.map(
-        (defaultOption) => `'${getOptionValueFromLabel(defaultOption.label)}'`,
+        (defaultOption) => `${getOptionValueFromLabel(defaultOption.label)}`,
```

Comment:
same thing here. We should not try to override the defaultValue from the input by re-adding quotes.